wiki/Astro-battery-optimisation.md

```diff
@@ -0,0 +1,55 @@
+# Astro: Battery optimisation
+
+Battery optimisation on Android is currently a mess, and the Astro is no exception.
+
+If you want an app to be able to run in the background; there are currently 3 places where you need to enable it. All of them are within the settings app.
+
+## Why
+
+Most apps don't give you any benefit from running in the background, while they make a massive difference to your battery usage. Yet there are a few use-cases that are really valuable. Eg:
```

Comment:
The only thing I'd like to add is, that some chat apps, use protocols like MQTT or Google's FCM, so they run in the background, but using a low-bandwidth, low-power pub/sub system, allowing for energy-efficient notifications.

What I would also add is that other ROMs (I know the Astro doesn't have other ROMs at this time), don't always come with Google FCM, if they're Degoogled.

Reply:
Done. Let me know if you're happy with the new version or if you'd like anything changed/refined.